### PR TITLE
Gui

### PR DESCRIPTION
**Pull request by DanudeSandstorm**
You didn't have a branch for gui. You'll have to create the branch and change the base branch.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -0,0 +1,59 @@
+-------------------------------------------
```
Comment (Mimickal) [inline README.txt]: Can you call this `dev setup.txt` or something like that?

File: build.gradle [hunk 2]
```diff
@@ -18,6 +18,7 @@ archivesBaseName = "floodgate"
 sourceCompatibility = targetCompatibility = "1.8" // Need this here so eclipse task generates correctly.
 compileJava {
     sourceCompatibility = targetCompatibility = "1.8"
+    options.compilerArgs << "-Xlint:unchecked"
```
Comment (Mimickal) [inline build.gradle:21]: What's this for?

File: src/main/resources/assets/floodgate/lang/en_US.lang [hunk 3]
```diff
@@ -1 +1,4 @@
-tile.floodgate.name=Floodgate
\ No newline at end of file
+tile.floodgate.name=Floodgate
+container.floodgate=Floodgate
+
+gui.block_breaker.enchanted_book.tooltip=Enchanted book goes here
```
Comment (Mimickal) [inline src/main/resources/assets/floodgate/lang/en_US.lang]: I assume this is part of the tutorial?

**Review by Mimickal — CHANGES_REQUESTED** [review]
I've created a `gui` branch in my repo. Can you re-open this PR against that branch?

File: src/main/resources/assets/floodgate/lang/en_US.lang (hunk 3 above)
Comment (DanudeSandstorm) [reply src/main/resources/assets/floodgate/lang/en_US.lang]: Aye, I left it in there so you'd see the tooltip on hover, I'll comment it out for now

File: build.gradle (hunk 2 above)
Comment (DanudeSandstorm) [reply build.gradle:21]: It is for debugging. The compiler was throwing a warning, and asked for the this option to explain the warning. It was due to "unsafe" casting.  I can remove this argument if you like

**Pull request merged**